_benchmarking/gen_fake_sqw_data.m

```diff
@@ -0,0 +1,129 @@
+function dataSource = gen_fake_sqw_data(dataInfo)
+%   This function will generate an sqw object for benchmarking using dummy_sqw
```

Comment:
after all the fake_*-->dummy_sqw changes, odd to leave fake here in the function name

Reply:
this one appears to have slipped through the cracks (I suspect because it started with gen_), will rename from fake_sqw to dummy_sqw